Project.toml

```diff
@@ -5,6 +5,8 @@ version = "0.12.0"
 
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+CUDSS = "45b445bb-4962-46a0-9369-b4df9d0f772e"
```

Comment:
I am not sure we should add CUDA and CUDSS in the direct dependencies. I would suggest moving them to the CUDA package extension.